### DISCUSSION TITLE
Where is the `IMPORT` function ?

### DISCUSSION THREAD

**Discussion by starrystarry9** [usememos/memos#3199] [Q&A] [answered]
I cannot find it alought I have done a search for all discussionds about `IMPORT`.

**Comment by denogio** [comment]
And the only place in the documentation mentioning import feature is the FAQ:

> [Can I migrate from other note-taking apps?](https://usememos.com/docs/faq#can-i-migrate-from-other-note-taking-apps)
Memos supports importing from various formats including Markdown files. Specific migration guides are available for popular platforms.

And nowhere to find anything on how you import.

**Reply by TheNomad11** [reply]
There is even a page where easy migration incl import is advertised https://usememos.com/features/import but without any explanations, I was also looking for it but could not find anything, it seems misleading communication as this feature is not existing and not planned at all according to https://github.com/usememos/memos/issues/5417 Searching the documentation for import gives no matches at all, export is not easy at all, so you are more less locked in

**Comment by ztm0929** [comment]
@denogio @TheNomad11 This feature is marked as `WIP` (work in progress) on the feature description page (too small😪), which means it is still under development. Please be patient.

By the way, the entire project is still in beta version(`v0.x.x`), so there may be some break changes to the code and documentation.

<img width="1234" height="723" alt="image" src="https://github.com/user-attachments/assets/c3a0cd1a-abad-4570-89eb-42b3015941ab" />

https://usememos.com/features

**Comment by bluedbird** [comment, marked as answer]
We're working on building import tools for Memos! If you're interested, drop by and let us know what app you're migrating from: https://github.com/orgs/usememos/discussions/5712

**Comment by vuongtuha** [comment]
I need mass import from json file. Wish it would be done soon.
